security/keycloak-webapp/src/test/java/io/quarkus/ts/openshift/security/keycloak/containers/KeycloakQuarkusTestResource.java

```diff
@@ -0,0 +1,52 @@
+package io.quarkus.ts.openshift.security.keycloak.containers;
+
+import io.quarkus.test.common.QuarkusTestResourceLifecycleManager;
+import org.testcontainers.containers.BindMode;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.wait.strategy.Wait;
+
+import java.time.Duration;
+import java.util.Collections;
+import java.util.Map;
+import java.util.Optional;
+
+public class KeycloakQuarkusTestResource implements QuarkusTestResourceLifecycleManager {
+
+    private static final String OIDC_AUTH_URL_PROPERTY = "quarkus.oidc.auth-server-url";
+
+    private static final String USER = "admin";
+    private static final String PASSWORD = "admin";
+    private static final String REALM = "test-realm";
+    private static final int PORT = 8080;
+
+    private static final String REALM_FILE = "/tmp/realm.json";
+    private static final String KEYCLOAK_IMAGE = "quay.io/keycloak/keycloak:11.0.1";
```

Comment:
`11.0.3` is the latest one

Reply:
Updated and checked that the test is still working fine.